tests/atlasgen/test_annotation_utils.py

```diff
@@ -12,27 +13,67 @@
 
 
 @pytest.mark.parametrize(
```

Comment:
I would suggest to split this test into three smaller tests, for clarity.

- One test for default (`acronym_length=1`)
- One test for non-default (`acronym_length=3` or similar)
- One for error case (`acronym_length = len(atlas_name)+1`)

instead of using `pytest.param`?